README.md

```diff
@@ -201,6 +201,12 @@ hostname_groups_enabled = true
 pc = ["example_group"]
 laptop = ["example_group"]
 server = ["example_group"]
+
+# Whether to default to installing cargo packages with --locked option.
+# This setting can be overriden on a per-package base using 
+# { systemwide = false|true }.
```

Comment:
I don't think you need to give an example for `per-package` override since that should already given in the example group file section. It's still nice to mention it though.

Reply:
Ok I will remove it anyway made a copy paste mistake..